src/components/HistoryTables/detail.tsx

```diff
@@ -150,10 +147,10 @@ export const MessageDetailBase = ({
 
   const confirmationCount = useMemo<number>(
     () =>
-      chainHeadSubscription.data?.chainHead.height && !!message?.height
-        ? chainHeadSubscription.data.chainHead.height - Number(message.height)
+      chainHeadSubscription.data?.chainHead.height && message.height
+        ? chainHeadSubscription.data.chainHead.height - message.height
         : 0,
-    [message?.height, chainHeadSubscription.data?.chainHead.height]
+    [message.height, chainHeadSubscription.data?.chainHead.height]
```

Comment:
Are you sure this doesn't cause an error on initial load?

Reply:
Message is passed as a mandatory prop in `MessageDetailBase`, so it should never render when `message` is null or undefined. In the return statement we were also assuming message to be defined, without the optional chaining. Seems to work fine.